src/ca/mcgill/cs/jetuml/gui/EditorFrame.java

```diff
@@ -445,18 +412,15 @@ public void addGraphType(String pResourceName, final Class<?> pGraphClass)
 		
 		aNewMenu.getItems().add(aAppFactory.createMenuItem(pResourceName, pEvent ->
 		{
-			SwingUtilities.invokeLater(() -> 
+			try 
 			{
-				try 
-				{
-					GraphFrame frame = new GraphFrame((Graph) pGraphClass.newInstance(), aTabbedPane);
-					addTab(frame);
-				}
-				catch (Exception exception) 
-				{
-					exception.printStackTrace();
-				}
-			});
+				GraphFrame frame = new GraphFrame((Graph) pGraphClass.newInstance(), aTabbedPane);
+				addTab(frame);
+			}
+			catch (Exception exception) 

```

Comment:
should be narrowed to precise type: `IllegalAccessException`. Please remove excess white space and lines. Replace `printStackTrace` with `assert false;`